core/model.md

```diff
@@ -680,21 +624,22 @@ The following describes the attributes of the Registry model:
 ### `groups.<STRING>.resources.<STRING>.hasdocument`
 - Type: Boolean (`true` or `false`, case-sensitive).
 - OPTIONAL.
-- Indicates whether or not Resources of this type can have a document
-  associated with it. If `false` then the xRegistry metadata becomes "the
-  document". Meaning, an HTTP `GET` to the Resource's URL will return the
-  xRegistry metadata in the HTTP body. The `xRegistry-` HTTP headers,
-  representing the Resource metadata, MUST NOT be used for requests or
-  response messages for these Resources. Use of `$details` on the request
-  URLs MAY be used to provide consistency with the cases where this
-  attribute is set to `true` - but the output remains the same.
+- Indicates whether or not Resources of this type have a domains-specific
+  document associated with it. If `false` then the xRegistry metadata becomes
+  "the document". Meaning, a query to the Resource's URL will return the
+  xRegistry metadata and not the domain-specific document.
 
   A value of `true` does not mean that these Resources are guaranteed to
-  have a non-empty document, and an HTTP `GET` to the Resource MAY return an
-  empty HTTP body.
+  have a non-empty document, and a query to the Resource MAY return an
+  empty document.
+
+  See
+  [Document Resources vs Metadata-Only Resources](./spec.md#document-resources-vs-metadata-only-resources)
+  for more information.
+
 - When not specified, the default value MUST be `true`.
-- A value of `true` indicates that Resource of this type supports a separate
-  document to be associated with it.
+- A value of `true` indicates that Resources of this type MUST a separate
+  document to be associated with it, even if it's empty.
```

Comment:
```suggestion
- A value of `true` indicates that each Resource of this type MUST have a separate
  document associated with it, even if it's empty.
```

(At the very least, some word after "MUST", but that on its own wasn't quite enough. Happy for others to wordsmith now I've drawn attention to it.)

Reply:
yup, the missing "have" was the real issue, but I made the other edits too